File: 03.Functions-And-Statements-More-Exercise/numberModification.js
function modifyNumber(number) {
    let numberAsArray = String(number).split('')
    let averageValue = checkAverageValue(numberAsArray)

    function checkAverageValue(num) {
        let totalValue = 0
        for (const number of num) {
            totalValue += Number(number)
        }

        return totalValue / numberAsArray.length
    }

    while (averageValue < 5) {
        numberAsArray.push('9')
        averageValue = checkAverageValue(numberAsArray)
    }

    console.log(numberAsArray.join(""))
}
    

modifyNumber(101)
modifyNumber(5835)

File: 03.Functions-And-Statements-More-Exercise/number_modification.py
def modify_number(number):

    def check_average_value(num):
        total_value = 0
        for n in num: 
            total_value += int(n)
        
        return total_value / len(number_as_list)
    
    number_as_string = str(number)
    number_as_list = list(number_as_string)
    average_value = check_average_value(number_as_list)


    while average_value < 5:
        number_as_list.append('9')
        average_value = check_average_value(number_as_list)

    print("".join(number_as_list))

modify_number(101)
modify_number(5835)